sites/gc-page-feedback/_base.scss

```diff
@@ -0,0 +1,31 @@
+/*
+  @title: Page feedback form - Base
+ */
+
+// In noscript/basic HTML mode...
+.no-js,
+.wb-disable {
+	.gc-pg-hlpfl {
+		// Left-align the legend and yes/no buttons
+		.text-left-nojs {
+			text-align: left;
+		}
+
+		// Disable row gutters (default selector isn't specific-enough)
+		.row-no-gutters {
+			[class*="col-"] {
+				padding-left: 0;
+				padding-right: 0;
+			}
+		}
+
+		// Make the legend full width
+		.col-sm-12-nojs {
+			@extend .col-sm-12;
+		}
+
+		.col-md-12-nojs {
+			@extend .col-md-12;
+		}
```

Comment:
This is interesting; could even be extracted from here and made something more generic to be reused in other components. @duboisp thoughts?

Reply:
I can't really think of anything off the top of my head that would benefit from those kinds of classes. But yeah, they could totally be turned into a set of generic utility classes if anyone can find other uses for them.

Reply:
Btw https://github.com/wet-boew/GCWeb/pull/1916#discussion_r982687664 (and similar selector clashes - if they exist) would need to be kept in mind for this.

Reply:
Something else worth noting is that regular ``col-*-#`` classes  can't override ``col-*-#-nojs`` ones when in in noscript/basic HTML modes.

For example, something like ``col-sm-12-nojs col-md-6`` would produce a 50% grid width in JS mode's medium/large views, whereas noscript/basic HTML modes would be 100%. That behaviour is due to the ``nojs`` classes having more-specific selector rules than the regular grid classes.

Anyway, it's not a problem in OPUFT's use case. But it'd need to be kept in mind if you try rolling out generic utility classes that ``@extend`` grid classes in the future.

Reply:
FYI I got rid of ``col-md-12-nojs`` since it isn't needed in practice.

I'll hold off marking this conversation as resolved in case @duboisp wants to chime in.